crates/sui-framework/packages/dwallet-system/sources/tendermint_lc.move

```diff
@@ -14,13 +13,28 @@ module dwallet_system::tendermint_lc {
     const TimestampInvalid: u64 = 5; 
     const NextValidatorsHashInvalid: u64 = 6; 
     const EUpdateFailed: u64 = 7;
-
+    
     // TODO: Make Client shareobject
     struct Client has key, store {
         id: UID,
-        latest_height: u64
+	chain_id: vector<u8>,
+	trust_threshold: u256,
+	trusting_period: u256,
+	clock_drift: u256,
+	latest_height: u64,
     }
 
+    public fun init_client(height: u64, chain_id: vector<u8>, trust_threshold: u256, trusting_period: u256, clock_drift: u256, ctx: &mut TxContext): Client {
+	Client {
+            id: object::new(ctx),
+	    chain_id: chain_id,
+            latest_height: height,
```

Comment:
something wrong with linting

Reply:
In `move language server,` they always assume the id from `sui::object::new.` However, in Ika, the id is created by the method `dwallet::object::new`. So, this is why they return a linting error here.